lib/msal-core/src/AuthResponse.ts

```diff
@@ -17,6 +18,7 @@ export type AuthResponse = {
     expiresOn: Date;
     account: Account;
     accountState: string;
+    claims: QPDict;
```

Comment:
If we already include these as part of account, is there any value in adding this to the response object?

Reply:
We can choose not to, it is to tie up request-response as most people look up for the response for claims. Account is one step further.

Reply:
I like having it on the response as well, otherwise people have to get the response, and then call get account vs just grabbing it.  

Can still get from account if its out of band from request

Reply:
the account object is already part of the response object, so they wouldn't have to call any other function. It would just be accessing the account object we return in the response.

Reply:
Line 18

Reply:
Resolved to both "Account" and "Response" having claims.